ntia-diagnostics.sigmf-ext.md

```diff
@@ -0,0 +1,178 @@
+| `28v_aux_powered`     |false| boolean | N/A            | Boolean indicating if the 28 volt auxillary power is on. |
+| `pwr_box_temp`        |false| double  | degree Celsius | Ambiant temperature at power distribution.               |
+| `pwr_box_humidity`    |false| double  | percent        | Humidity at power distribution.                          |
+| `rf_box_temp`         |false| double  | degree Celsius | Ambiant temperature around the signal analyzer.          |
+| `sigan_internal_temp` |false| double  | degree Celsius | Internal temperature reported by the signal analyzer     |
+
+### 0.4 The `Computer` Diagnostics Object
+
+The `Computer` diagnostics object has the following properties:
+
+| name               |required| type                            | unit           | description                                                                                             |
+|--------------------|--------------|---------------------------------|----------------|---------------------------------------------------------------------------------------------------------|
+| `cpu_min_clock`  |false| double                          | MHz            | Minimum sampled clock speed.                                                                            |
+| `cpu_max_clock`  |false| double                          | MHz            | Maximum sampled clock speed.                                                                            |
+| `cpu_mean_clock` |false| double                          | MHz            | Mean sampled clock speed.                                                                               |
+| `cpu_uptime`        |false| string                          |days| Number of days since the computer started                        |
+| `action_cpu_usage`        |false| double                          | percent        | CPU utilization during action execution.                                                                |
+| `system_load_5m`          |false| double                          | percent        | Number of processes in a runnable state over the previous 5 minutes as a percent of the number of CPUs. |
+| `memory_usage`     |false| double                          | percent        | Average percent of memory used during action execution.                                                  |
+| `cpu_overheating`      |false| boolean                         | N/A            | True if the CPU is overheating.                                                                             |
+| `cpu_temp`             |false| double                          | degree Celsius | CPU temperature.                                                                                   |
+| `scos_start`       |false| string                          |ISO-8601 (see above)| The time at which the SCOS API container started.                   |
```

Comment:
In the code this appears to be called `scos_start_time` instead of `scos_start` as shown here.